pyth-sdk/src/lib.rs

```diff
@@ -3,17 +3,59 @@ use borsh::{
     BorshSerialize,
 };
 
+use hex::FromHexError;
 use schemars::JsonSchema;
 
+pub mod utils;
+
 mod price;
 pub use price::Price;
 
-/// Unique identifier for a price.
-pub type PriceIdentifier = [u8; 32];
+#[derive(
+    Copy,
+    Clone,
+    Debug,
+    Default,
+    PartialEq,
+    Eq,
+    BorshSerialize,
+    BorshDeserialize,
+    serde::Serialize,
+    serde::Deserialize,
+    JsonSchema,
+)]
+#[repr(C)]
+pub struct Identifier(
+    #[serde(with = "hex")]
+    #[schemars(with = "String")]
```

Comment:
It might also be nice to have the debug/display representation of Identifiers be the hex string representation.

Reply:
Good idea!